libs/ui/packages/react/src/pre-ldls/components/VirtualList/VirtualList.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { ReactNode } from "react";
+import { FixedSizeList as List } from "react-window";
+import AutoSizer from "react-virtualized-auto-sizer";
+
+export const VirtualList = ({
+  count,
+  itemHeight,
+  renderRow,
+}: {
+  count: number;
+  itemHeight: number;
+  renderRow: (index: number) => ReactNode;
```

Comment:
```suggestion
  renderItem: (index: number) => ReactNode;
```
Just to be consistent